policyengine_us/parameters/gov/states/al/tax/income/exemptions/dependent.yaml

```diff
@@ -22,6 +22,8 @@ metadata:
   type: single_amount
   label: Alabama dependent exemption amount
   reference:
+  - title: 2023 Alabama Department of Revenue -Form 40 Booklet
+    href: https://www.revenue.alabama.gov/ultraviewer/viewer/basic_viewer/index.html?form=2024/01/23f40bk.pdf#page=9
```

Comment:
I think this should be page 10, could you double check all the references that the pages are correct?